examples/accelerate_configs/deepspeed_zero2.yaml

```diff
@@ -2,7 +2,6 @@ compute_environment: LOCAL_MACHINE
 debug: false
 deepspeed_config:
   deepspeed_multinode_launcher: standard
-  gradient_accumulation_steps: 1
```

Comment:
why this has been removed?

Reply:
We can specifiy the `gradient_accumulation_steps` via `TrainingArguments` and accelerate will handle that automatically! Hard-coding `gradient_accumulation_steps` here I think causes issues for training if we do `--gradient_accumulation_steps 16`.

Reply:
makes sense, thanks for explaining !